packages/main-api/src/controllers/v3/teams/groups/create_group.rs

```diff
@@ -99,8 +100,15 @@ pub async fn create_group_handler(
         .execute(&dynamo.client)
         .await?;
 
+    // Extract just the UUID from EntityType::TeamGroup(uuid)
+    let group_uuid = if let EntityType::TeamGroup(uuid) = &group_sk {
+        uuid.clone()
+    } else {
+        group_sk.to_string()
+    };
+
     Ok(Json(CreateGroupResponse {
         group_pk: group_pk.to_string(),
-        group_sk: group_sk.to_string(),
+        group_sk: group_uuid,
```

Comment:
```suggestion
        group_sk: group_sk.to_string(),
```

Instead of including the UUID in the response, send the Partition Key or EntityType itself.